packages/jest-worker/src/index.ts

```diff
@@ -72,18 +74,23 @@ export default class JestWorker {
   private _farm: Farm;
   private _options: FarmOptions;
   private _workerPool: WorkerPoolInterface;
+  private _inspector: Session | undefined;
 
   constructor(workerPath: string, options?: FarmOptions) {
     this._options = {...options};
     this._ending = false;
 
+    this._inspector = this.setUpInspector();
+
     const workerPoolOptions: WorkerPoolOptions = {
-      enableWorkerThreads: this._options.enableWorkerThreads ?? false,
-      forkOptions: this._options.forkOptions ?? {},
-      maxRetries: this._options.maxRetries ?? 3,
-      numWorkers: this._options.numWorkers ?? Math.max(cpus().length - 1, 1),
+      enableWorkerThreads: this._options.enableWorkerThreads || false,
```

Comment:
why the `??` to `||` changes?

Reply:
Oh.. I didn't pay enough attention while I was doing my rebase.